src/playerClass.ts

```diff
@@ -37,6 +39,14 @@ export class Player {
     return alivePokemonsTab;
   }
 
+  reduceNumberMango(): void {
```

Comment:
please rename this method :P and add validation inside:
if you ate mango once, then calling this method again should throw an error